database_admin/migrations/066_package_latest_redefine.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP FUNCTION IF EXISTS refresh_latest_packages_view;
+DROP INDEX IF EXISTS package_latest_cache_pkey;
+DROP MATERIALIZED VIEW IF EXISTS package_latest_cache;
```

Comment:
If we run the down migration, it'll delete the view and the refresh task will fail. I'd leave the down migration empty